app/(main)/(screens)/questions.tsx

```diff
@@ -1,18 +1,27 @@
-import React, { useState } from "react";
-import { Pressable } from "react-native";
+import React, { useEffect, useState } from "react";
+import { Pressable, Image } from "react-native";
 import { useTheme } from "@react-navigation/native";
 import { SURVEY_DATA } from "@/constants/SurveyData";
-import {
-  submitSurvey,
-} from "@/components/surverHandlers";
+import { submitSurvey } from "@/components/surverHandlers";
 import { stylesFn } from "@/styles/Questions.styles";
 import { useRouter } from "expo-router";
 import Colors from "@/constants/Colors";
 import { SurveyAnswer } from "@/types/Survey";
 import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
-import { faArrowRight, faChevronLeft } from "@fortawesome/free-solid-svg-icons";
+import {
+  faArrowRight,
+  faCheck,
+  faChevronLeft,
+} from "@fortawesome/free-solid-svg-icons";
 import Select from "@/components/ui/select";
 import { Text, View } from "@/components/theme/Themed";
+import { useSocialContext } from "@/contexts/social-context.provider";
```

Comment:
We can shift this to the contexts folder index.ts file and import it from there to maintain consistency.